README.md

```diff
@@ -68,6 +69,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+I think we should call strip_punctuation inside of the analyze_mood method, anytime before the .each loop so the loop doesn't analyze any words that still have the punctuation attached.
```

Comment:
Good. But why did you call it inside analyze_mood instead of outside it? (I agree with your choice, I just want to know your reasoning.)